### PR TITLE
Document sensuctl edit, clarify sensuctl create

### LINKED ISSUE #763: sensuctl reference documentation should describe support for multiple resources per input file
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)

## Expected Behavior

sensuctl reference documentation should describe supported use cases for resource creation

## Current Behavior

`sensuctl create` subcommand permits defining multiple resources in a single input file, but this isn't explicitly documented.

See https://gist.github.com/cwjohnston/77a4a9f18e5cc19b5de09d9f528c0f1c for an example file which creates multiple resources when passed to `sensuctl create -f`

### LINKED ISSUE #913: Document sensuctl edit
Add documentation for updating resource definitions with sensuctl edit to the sensuctl reference doc

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Document sensuctl edit
- Clarify creating multiple resources with sensuctl create
- Fix typo in handler definition example

## Motivation and Context

Closes #763 
Closes #913

### CODE REVIEW

File: content/sensu-go/5.0/sensuctl/reference.md [hunk 1]
```diff
@@ -297,12 +307,16 @@ an example, and [this table][3] for a list of supported types).
...
 {{< /highlight >}}
 
-Write all checks to `my-resources.json` in `wrapped-json` format:
+_NOTE: Commas cannot be included between resource definitions when using `sensuctl create -f`._
+
+To create all resources in `wrapped-json` format from `my-resources.json` using `sensuctl create`:
+
 {{< highlight shell >}}
-sensuctl check list --format wrapped-json > my-resources.json
+sensuctl create -f my-resources.json
```
Comment (Blue0ctober) [inline content/sensu-go/5.0/sensuctl/reference.md]: We should use long form flags when possible vs short flags. 

```suggestion
sensuctl create --file my-resources.json
```

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
Added a comment for changing one of the commands

File: content/sensu-go/5.0/sensuctl/reference.md [hunk 2]
```diff
@@ -297,12 +307,16 @@ an example, and [this table][3] for a list of supported types).
 }
 {{< /highlight >}}
 
-Write all checks to `my-resources.json` in `wrapped-json` format:
+_NOTE: Commas cannot be included between resource definitions when using `sensuctl create -f`._
```
Comment (apaskulin) [inline content/sensu-go/5.0/sensuctl/reference.md]:
```suggestion
_NOTE: Commas cannot be included between resource definitions when using `sensuctl create --file`._
```

File: content/sensu-go/5.0/sensuctl/reference.md [hunk 3]
```diff
@@ -319,6 +333,19 @@ cat my-resources.json | sensuctl create
...
+Sensuctl allows you to edit resource definitions using `sensuctl edit`.
+To use `sensuclt edit`, specify the resource [type][3] and resource name.
+
+For example, to edit a handler named `slack` using `sensuctl edit`:
+
+{{< highlight shell >}}
+sensuctl edit handler slack
+{{< /highlight >}}
+
+Sensuctl resource editing is available in YAML format only.
```
Comment (apaskulin) [inline content/sensu-go/5.0/sensuctl/reference.md]:
```suggestion
```

**Pull request merged**